_loadMonkey.js

```diff
@@ -1,12 +1,17 @@
 /* global  loadKeyEvents, toastr, loadTypeAhead, displayHelp, displayStartup, loadJumpToDatePicker */
 
+const ignoredFeatures = typeof window.ignoredFeatures !== 'undefined' ? window.ignoredFeatures : []; 
```

Comment:
Needed to attach the variable to the window in order to read it. But was able to confirm on my personal db:
![image](https://user-images.githubusercontent.com/7143571/90708430-cc944780-e267-11ea-90f7-28d64acb8e1f.png)

Confirmed that in this example that the daily note popup was disabled but other monkey features worked.

As we discussed on slack, I'd be happy to do the next iteration of this by using a Roam page for configuration. Any opinions on page name and format?